twa

```diff
@@ -2,6 +2,14 @@
 
 # twa: a tiny website auditing script
 
+
+# Colored output
+GREEN='\033[0;32m'
```

Comment:
Prefix these with `COLOR_`, e.g. `COLOR_BLUE` and `COLOR_RESET` (`NC` isn't accurate; sending `\033[0m` will cause all current attributes to reset. There's `\033[39;49m`, but fewer terminals support it).